.github/workflows/_cve-remediation.yml

```
@@ -0,0 +1,153 @@
+    permissions:
+      contents: read
+    outputs:
+      has_issues: ${{ steps.discover.outputs.has_issues }}
+      issues: ${{ steps.discover.outputs.issues }}
+      issue_count: ${{ steps.discover.outputs.issue_count }}
+      total_issue_count: ${{ steps.discover.outputs.total_issue_count }}
+    steps:
+      - name: Checkout reusable workflow assets
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ job.workflow_repository }}
```

actionlint (1.7.12)

[error] property "workflow_repository" is not defined in object type {check_run_id: number; container: {id: string; network: string}; services: {string => {id: string; network: string; ports: {string => string}}}; status: string}

(expression)